backup.go

```diff
@@ -92,17 +95,122 @@ func (backup *Backup) GetKeys() ([]string, error) {
 	return result, nil
 }
 
-func (backup *Backup) fetchSentinel() (sentinelDto S3TarBallSentinelDto, err error) {
+func (backup *Backup) fetchSentinel() (S3TarBallSentinelDto, error) {
+	sentinelDto := S3TarBallSentinelDto{}
 	backupReaderMaker := NewS3ReaderMaker(backup.Folder, backup.getStopSentinelPath())
 	backupReader, err := backupReaderMaker.Reader()
 	if err != nil {
-		return S3TarBallSentinelDto{}, err
+		return sentinelDto, err
```

Comment:
m.b. just return nil?

Reply:
It's a structure, so it's impossible to return nil here. Also, when I tried to do it in such a way I got several issues with value/reference type, in this function usages.